src/app/sidebar/components/CoursesList.tsx

```diff
@@ -8,25 +8,34 @@ import { Card } from './Card';
 
 type CoursesListProps = {
   term: string;
+  route: string;
```

Comment:
Instead of passing route as a prop, you can do something like [this](https://github.com/VikeLabs/clockwork/blob/main/src/app/header/components/NavButtons.tsx#L6)